src/bin/vip-import-sql-status.js

```diff
@@ -25,34 +25,49 @@ environments{
 }
 `;
 
+const usage = 'vip import sql status';
+
+// Command examples
+const examples = [
+	{
+		usage: 'vip @example-app.develop import sql status',
+		description:
+			'Check the status of the most recent SQL database file import to the develop environment of the "example-app" application.\n' +
```

Comment:
```suggestion
			'Check the status of the most recent SQL database import to the develop environment of the "example-app" application.\n' +
```

The action the user is taking is an "SQL database import" - the file is just the means to move that data from their local machine to the VIP application's database.

Reply:
Agreed!